### DISCUSSION TITLE
Auto rename closing tag

### DISCUSSION THREAD

**Discussion by dmeffe-rw** [phcode-dev/phoenix#1693] [Q&A] [answered]
Is there a way to automatically change the closing tag when a tag is changed? either a shortcut, setting, or a plugin?

**Comment by abose** [comment]
There is a `Changing tags` extension, but it doesnt have the best UX. We will be adding native support for tag sync editing in this release or the next.

**Comment by abose** [comment, marked as answer]
Done as part of https://github.com/phcode-dev/phoenix/pull/1765  . This feature will be available in the August-Mid release.
